Sources/HNBUI/TabBar.swift

```diff
@@ -123,10 +148,23 @@ public final class TabBar: UIView {
             hStack.addArrangedSubview(tabBarItem)
         }
         if items.count > 0 {
-            selectedItem = items[0]
+            layoutIfNeeded()
```

Comment:
`layoutIfNeeded()` is needed to layout `hStack.arrangedSubviews`.
subviews are needed to layout for `underBarView` to set its constraints to subviews' anchors.